### PR TITLE
Add CI setting

### PR DESCRIPTION
**Pull request by chaspy**

### CODE REVIEW

**Comment by chaspy** [comment]
I think rarely have the chance to update this repository... but I'll improve 😄 
please review and merge. @quipper/sre

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          command: docker build -t quipper/docker-ansible:`date "+%Y%m%d"` .
+      - run:
+          command: |
+            if [[ $CIRCLE_BRANCH == "master" ]]; then
+              docker login -u ${DOCKERHUB_USER} -p ${DOCKERHUB_PASSWORD}
+              docker push quipper/docker-ansible:`date "+%Y%m%d"`
```
Comment (yuya-takeyama) [inline .circleci/config.yml]: `date "+%Y%m%d"` can be varied across the steps when it started around 0:00.
So it should be a variable or something?

Comment (chaspy) [reply .circleci/config.yml]: ok, I'll use commit hash(`CIRCLE_SHA1`). thanks!

**Comment by chaspy** [comment]
please review and merge.

and, if this is merged, should I fix this to generated commit hash?
  https://github.com/quipper/server-templates/blob/master/circle.yml#L9
(it is not better to specify the latest tag?)

**Comment by yuya-takeyama** [comment]
> (it is not better to specify the latest tag?)

Sounds good to me 👍

**Pull request merged**